edgePy/DGEList.py

```diff
@@ -99,6 +107,33 @@ def __init__(
                     "and samples must be present"
                 )
 
+    def copy(
```

Comment:
What's the rationale behind "changing" attributes during the copy? I think this is confusing and copy() should just do what is suggests (copy)

Reply:
It only changes the attributes if you need them changed.  You can used .copy() and it will do exactly what you expect.